compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -503,6 +504,70 @@ Flow::wrapOpInDispatchRegion(RewriterBase &rewriter, Operation *op) {
   return newRegionOp;
 }
 
+bool Flow::isDequantizationLikeOp(Operation *op) {
```

Comment:
This is the basic change from this PR. The previous dequantization matching has been replaced with this matcher.

Reply:
This is not a trivial function, can you add a doc to it?

Reply:
I have some comments in RegionOpUtils.h. Should I add something here as well?

Reply:
oops, I missed that part. We don't need the comment in `.cpp`, thank you!